code/modules/projectiles/updated_projectiles/guns/specialist.dm

```diff
@@ -764,9 +764,9 @@
 	log_explosion("[usr] fired the [src] at [AREACOORD(loc)].")
 
 /obj/item/weapon/gun/launcher/rocket/wield(mob/living/user)
-	. = ..()
-	if(user.mind?.cm_skills && user.mind.cm_skills.spec_weapons < 0)
-		do_after(user, 15, TRUE, 5, BUSY_ICON_HOSTILE)
```

Comment:
s the removal of the icon intended here?

Reply:
Wielding will need quite a bit of a clean up in general. no visible message too (keep in mind the user will still see the prog bar.